### DISCUSSION TITLE
Oil Palm model- not responding for irrigation and fertilizer rates

### DISCUSSION THREAD

**Discussion by KrishnaDevkota** [APSIMInitiative/ApsimX#8671] [General]
Dear Colleagues,
I posted this question three weeks ago also, did not receive any reply. 
I am really struggling to run Oil Palm model. When I changed the climate file, it did not respond. It did not respond to the irrigation amount and fertilizer rates as well. I am not sure which irrigation or fertilizer module it accepts; I checked all combinations so far. Can anybody please kindly help me? I really appreciate your help and I really need to run this model.
Best,
Krishna

**Comment by hut104** [comment]
Hi Krishna,
Thank you for reaching out here.  It might be useful for you to upload any files required to reproduce this issue.  This would allow some of our user community to assist in diagnosing the issue here.

**Reply by KrishnaDevkota** [reply]
Dear Neil, 
Thank you very much for your feedback. Herewith I have attached my files in a zip folder (6 met files and one apsimx). 
[KrishnaDatePalmFiles.zip](https://github.com/APSIMInitiative/ApsimX/files/14367438/KrishnaDatePalmFiles.zip)
The climate files are from UAE, the crop is fully irrigated. Actually, I want to adopt oil palm model to simulate date palm. Per tree fruit yield is 50-125 kg/year and plant density 150-225/ha. 
I would really appreciate if APSIM community can assist me in this regard. 
Best, 
Krishna

**Comment by KrishnaDevkota** [comment]
Dear Neil (@hut104 ), Dear Dean (@hol353 ), and Apsim Developer Colleagues,
I encountered a problem while running the OilPalm model. The soil profile is not picking applied water and nitrogen fertilizer (it has been applied and appears as applied in the output file) in APSIM NG, resulting in carbon stress and no yield. 
When I run the same model in APSIM Classic, it generates a yield with the same files and inputs.  It looks like there is a bug or something is missing while transferring in NG. 
I posted this request two times already also but did not get any support so far. Can you please kindly provide your support in fixing this? Attached herewith my latest simulation files. 
[2ndTimeOilPalm.zip](https://github.com/APSIMInitiative/ApsimX/files/14396393/2ndTimeOilPalm.zip)

Best,

**Comment by KrishnaDevkota** [comment]
Dear Nil, @hut104, thank you very much. Now OilPalm model runs well under APSIMx. However, it is still creating an error in N allocation; attached snapshot. May I request you/developer team please check and fix it? Thank you so much!
<img width="914" alt="N distribution error" src="https://github.com/APSIMInitiative/ApsimX/assets/29770777/69e017e2-e9c5-4440-9460-2e95e85c3ece">
